graspit-core.pro

```diff
@@ -148,6 +149,10 @@ SOURCES	+= src/arch.cpp \
 	src/worldElementFactory.cpp \
 	src/math/matrix.cpp \
 	src/Collision/collisionInterface.cpp \
+        src/dynamics/dynamics.cpp \
+        src/dynamics/bulletDynamics.cpp \
```

Comment:
I think the bulletDynamics files (both .h and .cpp) should be included here only if they are actually used - same for the graspitDynamics files. They should be guarded by bullet_dyamics{...} or graspit_dynamics{...} respectively.